File: OfflineEveBrowser.php
<?php
require_once('IEveBrowser.php');

class OfflineEveBrowser implements IEveBrowser {
   public function IsIGB() {
      return 1;
   }

   public function IsTrusted() {
      return 1;
   }

   public function RequireTrust() {
   }

   public function RegionName() {
      return 'Delve';
   }

   public function ConstellationName() {
      return "W-4U1E";
   }

   public function SolarSystemName() {
      return "QY6-RK";
   }

   public function AllianceID() {
      return 824518128;
   }

   public function AllianceName() {
      return "GoonSwarm";
   }

   public function CharacterID() {
      return 1164427832;
   }

   public function CharacterName() {
      return "WoogyDude";
   }

   public function CorporationID() {
      return 749147334;
   }

   public function CorporationName() {
      return "GoonFleet";
   }

   public function StationName() {
      return "None";
   }

   public function Server() {
      return "87.237.38.200:26000";
   }

   public function CorporationRole() {
      return 2199024312320;
   }

   public function NearestLocation() {
      return "QY6-RK VI - Moon 24";
   }
}

?>

File: App_Code/OfflineEveBrowser.cs
using System;
using System.Data;
using System.Configuration;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using System.Net;

/// <summary>
/// Basicly a mock object for the EVE Browser to be used when testing outside of the EVE IGB.
/// </summary>
public class OfflineEveBrowser : IEveBrowser
{
    public OfflineEveBrowser()
    {
    }

    public bool IsIGB
    {
        get { return true; }
    }

    public bool IsTrusted
    {
        get { return true; }
    }

    public bool RequireTrust()
    {
        return true;
    }

    public string RegionName
    {
        get { return "Delve"; }
    }

    public string ConstellationName
    {
        get { return "W-4U1E"; }
    }

    public string SolarSystemName
    {
        get { return "QY6-RK"; }
    }

    public long AllianceID
    {
        get { return 824518128; }
    }

    public string AllianceName
    {
        get { return "GoonSwarm"; }
    }

    public int CharacterID
    {
        get { return 1164427832; }
    }

    public string CharacterName
    {
        get { return "WoogyDude"; }
    }

    public int CorporationID
    {
        get { return 749147334; }
    }

    public string CorporationName
    {
        get { return "GoonFleet"; }
    }

    public string StationName
    {
        get { return "None"; }
    }

    public IPEndPoint Server
    {
        get { return new IPEndPoint(IPAddress.Parse("87.237.38.200"), 26000); }
    }

    public long CorporationRole
    {
        get { return 2199024312320; }
    }

    public string NearestLocation
    {
        get { return "QY6-RK VI - Moon 24"; }
    }
}
